env/env.js

```diff
@@ -45,7 +45,7 @@ const base = removeNullValues({
   REACT_APP_FEEDBACK_ADDR: 'feedback@votebymail.io',
   REACT_APP_ELECTION_OFFICIAL_ADDR: 'elections@votebymail.io',
   REACT_APP_SUPPORT_ADDR: 'support@votebymail.io',
-  MG_DOMAIN: 'email.votebymail.io',
+  MG_DOMAIN: 'email.dev.votebymail.io',
   MG_FROM_ADDR: 'Vote by Mail Application <application@email.votebymail.io>',
```

Comment:
You should change the `MG_FROM_ADDR` as well.  Otherwise, it might not work / doesn't do much.